README.md

```diff
@@ -515,6 +508,36 @@ CMD parameters:
 The contents of extracted package can be cleaned up using configured VLT rules by chaining [aemClean](#task-aemclean) task 
 `gradlew :aemDownload :aemClean`
 
+### Deleted package plugin tasks
```

Comment:
such info will not be included in docs but only in upgrade notes.

docs should only be about current GAP version.